qa/pull-tester/rpc-tests.sh

```diff
@@ -34,163 +34,163 @@ done
 #Run the tests
 
 testScripts=(
-  'paymentdisclosure.py'
-  'prioritisetransaction.py'
-  'wallet_treestate.py'
-  'wallet_protectcoinbase.py'
-  'wallet_shieldcoinbase.py'
-  'wallet_mergetoaddress.py'
-  'wallet_mergetoaddress_2.py'
-  'wallet.py'
-  'wallet_nullifiers.py'
-  'wallet_1941.py'
-  'wallet_grothtx.py'
-  'listtransactions.py'
-  'mempool_resurrect_test.py'
-  'txn_doublespend.py'
-  'txn_doublespend.py --mineblock'
-  'getchaintips.py'
-  'rawtransactions.py'
-  'rest.py'
-  'mempool_spendcoinbase.py'
-  'mempool_coinbase_spends.py'
-  'mempool_tx_input_limit.py'
-  'httpbasics.py'
-  'zapwallettxes.py'
-  'proxy_test.py'
-  'merkle_blocks.py'
-  'fundrawtransaction.py'
-  'signrawtransactions.py'
-  'walletbackup.py'
-  'key_import_export.py'
-  'nodehandling.py'
-  'reindex.py'
-  'decodescript.py'
-  'disablewallet.py'
-  'zcjoinsplit.py'
-  'zcjoinsplitdoublespend.py'
-  'zkey_import_export.py'
-  'getblocktemplate.py'
-  'bip65-cltv-p2p.py'
-  'bipdersig-p2p.py'
-  'nulldata.py'
-  'blockdelay.py'
-  'blockdelay_2.py'
-  'z_sendmany.py'
-  'sc_create.py'
-  'sc_split.py'
-  'sc_invalidate.py'
-  'sc_cert_base.py'
-  'sc_cert_nonceasing.py'
-  'sc_cert_fee.py'
-  'sc_cert_epoch.py'
-  'sc_cert_invalidate.py'
-  'sc_fwd_maturity.py'
-  'sc_rawcertificate.py'
-  'getunconfirmedtxdata.py'
-  'sc_cr_and_fw_in_mempool.py'
-  'sc_cert_change.py'
-  'sc_cert_orphans.py'
-  'sc_cert_maturity.py'
-  'sbh_rpc_cmds.py'
-  'sc_cert_ceasing.py'
-  'sc_cert_customfields.py'
-  'sc_cert_getraw.py'
-  'sc_quality_mempool.py'
-  'sc_ft_and_mbtr_fees.py'
-  'sc_ft_and_mbtr_fees_update.py'
-  'sc_bwt_request.py'
-  'sc_cert_quality_wallet.py'
-  'ws_messages.py'
-  'ws_getsidechainversions.py'
-  'sc_cert_ceasing_split.py'
-  'sc_async_proof_verifier.py'
-  'sc_quality_blockchain.py'
-  'sc_quality_voiding.py'
-  'sc_csw_actcertdata.py'
-  'sc_csw_actcertdata_null.py'
-  'sc_cert_ceasing_sg.py'
-  'sc_csw_nullifier.py'
-  'sc_getscinfo.py'
-  'sc_quality_nodes.py'
-  'sc_cert_memcleanup_split.py'
-  'sc_csw_fundrawtransaction.py'
-  'sc_proof_verifier_low_priority_threads.py'
-  'subsidyhalving.py'
-  'cbh_rpfix.py'
-  'cbh_rpcheck.py'
-  'tlsprotocols.py'
-  'mempool_double_spend.py'
-  'getblockmerkleroots.py'
-  'sc_block_partitions.py'
-  'sc_cert_bwt_amount_rounding.py'
-  'sc_csw_eviction_from_mempool.py'
-  'sc_csw_memcleanup_split.py'
-  'sc_csw_balance_exceeding.py'
-  'sc_stale_ft_and_mbtr.py'
-  'sc_cert_getblocktemplate.py'
-  'sc_cert_bt_immature_balances.py'
-  'sc_rpc_cmds_fee_handling.py'
-  'sc_cert_listsinceblock.py'
-  'sc_cert_dust.py'
-  'sc_keyrot.py'
-  'txindex.py'
-  'getblockexpanded.py'
-  'sc_rpc_cmds_json_output.py'
-  'sc_version.py'
-  'sc_getscgenesisinfo.py'
-  'fundaddresses.py'
-  'sc_getcertmaturityinfo.py'
-  'sc_big_commitment_tree.py'
-  'sc_big_commitment_tree_getblockmerkleroot.py'
+  'paymentdisclosure.py',99
```

Comment:
What about using a more "complex" separator (possibly using 2 or 3 characters) rather than a single comma?
Actually I can see current separator problematic only in case of test file name including comma (I would say probability 0.000...0001, but you never know), or maybe some other edge cases not being considered right now.